src/codeblock.js

```diff
@@ -103,7 +103,8 @@ export default class CodeBlock extends Plugin {
  *
  *		<pre><code class="language-javascript">window.alert( 'Hello world!' )</code></pre>
  *
- * You can customize the CSS class by specifying an optional `class` property in the language definition:
+ * You can customize the CSS class by specifying an optional `class` property in the language definition.
+ * You can set **multiple classes** but **only the first one** will be used as defining language class:
```

Comment:
I think we should also mention this in the guide https://ckeditor.com/docs/ckeditor5/latest/features/code-blocks.html#configuring-code-block-languages.